jkube-kit/jkube-kit-wildfly-jar/src/main/java/org/eclipse/jkube/wildfly/jar/enricher/WildflyJARHealthCheckEnricher.java

```diff
@@ -100,73 +141,17 @@ public void visit(ContainerBuilder containerBuilder) {
         super.create(platformMode, builder);
     }
 
-    private Probe discoverWildflyJARHealthCheck(boolean isReadiness) {
-
-        if (isAvailable()) {
-            Integer port = getPort();
-            if (port <= 0) {
-                return null;
-            }
-            int initialDelay = isReadiness ? getReadinessInitialDelay() : getLivenessInitialDelay();
-            // scheme must be in upper case in k8s
-            String scheme = getScheme().toUpperCase();
-            String path = isReadiness ? getReadinessPath() : getLivenessPath();
-
-            return new ProbeBuilder()
-                    .withNewHttpGet().withNewPort(port).withPath(path).withScheme(scheme).endHttpGet()
-                    .withFailureThreshold(getFailureThreshold())
-                    .withSuccessThreshold(getSuccessThreshold())
-                    .withInitialDelaySeconds(initialDelay).build();
-        }
-        return null;
-    }
-
     private boolean isAvailable() {
-        if (isProbeEnforced()) {
+        if (asBoolean(getConfig(Config.ENFORCE_PROBES))) {
             return true;
         }
         JavaProject project = getContext().getProject();
-        Plugin plugin = JKubeProjectUtil.getPlugin(project, BOOTABLE_JAR_GROUP_ID, BOOTABLE_JAR_ARTIFACT_ID);
+        Plugin plugin = JKubeProjectUtil.getPlugin(project, "org.wildfly.plugins", "wildfly-jar-maven-plugin");
```

Comment:
I'm not sure if you interpreted this from a comment in one of the tests.

**Production code** should be using the constants, that's not a problem.

However, the test code SHOULD NOT use the constants, especially if they are the same as the ones in the production code. The reason is simple, a change in the constant, will not make the test fail even though we've completely changed the specification constraint. Considering a test is used to verify a specification or a defined behavior, we should use either a literal, or if needed a variable defined in the same test (however, this might make the test less readable).

Reply:
Got it